pkg/constants/constants.go

```diff
@@ -18,9 +18,10 @@
 package constants
 
 const BrokerClusterPrefix = "broker-cluster-"
-const MasterBrokerContainerNamePrefix = "broker-master"
-const ReplicaBrokerContainerNamePrefix = "broker-replica"
+const BrokerContainerName = "broker"
 const AdminToolDir = "/home/rocketmq/rocketmq-4.5.0/bin/mqadmin"
```

Comment:
can not use for 4.6.0 ？

Reply:
We will change the admin tool directory name to a version-free name later.